pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -306,6 +306,45 @@ def test_to_datetime_tz_psycopg2(self):
                                     dtype='datetime64[ns, UTC]')
         tm.assert_index_equal(result, expected)
 
+    @pytest.mark.parametrize("box", [True, False])
```

Comment:
this test is ok, but rather I would like to see a parametrize on pretty much every other test function in this file. that tests both ``use_cache=True/False``